src/Internal/Service/PaymentFraudPreventionService.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
+ * Class PaymentFraudPreventionService
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Internal\Service;
+
+use WC_Payments_Account;
+
+/**
+ * Class PaymentFraudPreventionService
+ */
+class PaymentFraudPreventionService {
+
+	const TOKEN_NAME = 'wcpay-fraud-prevention-token';
+
+	/**
+	 * SessionService instance.
+	 *
+	 * @var SessionService
+	 */
+	private $session_service;
+
+	/**
+	 * Instance of WC_Payments_Account.
+	 *
+	 * @var WC_Payments_Account
+	 */
+	private $legacy_account_service;
```

Comment:
Within the payment process so far, we have not used `legacy_`, unless it's the "same" service, ex. `legacy_service` within `WCPay\Internal\Service\OrderService`. Any reason to explicitly mention it here?

Reply:
Nothing specific, I will update it.